ar_exercise/exercise.rb

```diff
@@ -31,41 +34,91 @@ class Apartment < ActiveRecord::Base
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
+# Before running pry, run file through ruby (fil)
+
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+all_tenants = Tenant.first
+
 # get all tenants older than 65
+all_tenants = Tenant.where("age: > ?" "65")
+
 # get all apartments whose price is greater than $2300
+all_apartments = Apartment.where("age: > ?" "2300")
+
 # get the apartment with the address "6005 Damien Corners"
+all_apartments = Apartment.where(address:"6005 Damien Corners")
+
 # get all tenants in that apartment
+all_tenants = Tenant.where(apartment_id: 6)
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+Tenant.all.each do |tenant|
+  puts "Name: #{tenant.name}, Id: #{tenant.id}"
+end
 # Iterate over each apartment, for each apartment, display it's address and rent price
+Apartment.all.each do |apartment|
+  puts "Address: #{apartment.address}, Rent: $#{apartment.monthly_rent}"
+end
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+Apartment.all.each do |apartment|
+  puts apartment.address
+  apartment.tenants.each do |tenant|
+    puts tenant.name
+  end
+  puts "-" * 50
+end
 
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
-# Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
+# Hint, the following methods will help: `new`, `create`, `save`, `update`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+the_bat_cave = Apartment.create(address: "123 Main St", monthly_rent: 2000, sqft: 600, num_beds: 2, num_baths: 1)
+# ID 22
+the_sinatra = Apartment.create(address: "1094 Bergen St", monthly_rent: 1800, sqft: 700, num_beds: 2, num_baths: 1)
+# ID 23
+the_wu_tang = Apartment.create(address: "2970 Flatbush Ave", monthly_rent: 1100, sqft: 650, num_beds: 1.5, num_baths: 1)
+# ID 24
+the_jeter = Apartment.create(address: "825 Fulton St", monthly_rent: 1300, sqft: 480, num_beds: 1, num_baths: 1)
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+margaret = Tenant.create(name: "Margaret", age: 24, gender: "Female", apartment_id: 22)
```

Comment:
It's best never to hard code numerical IDs when using active record.

Instead I'd do something like this:

```
margaret = Tenant.create(name: "Margaret", age: 24, gender: "Female", apartment: the_bat_cave)
```